public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -183,7 +183,7 @@ export function DashList(props: PanelProps<Options>) {
                 {dash.folderTitle && <div className={css.dashlistFolder}>{dash.folderTitle}</div>}
               </div>
               <IconToggle
-                aria-label={`Star dashboard "${dash.title}".`}
+                aria-label={`${dash.isStarred ? 'Unstar' : 'Star'} dashboard "${dash.title}"`}
```

Comment:
So, this is a really interesting time where we want the tooltip and the aria-label to differ because the experience changes depending on if you are using assistive technology or not.

If you think for a sighted user looking at the tooltip they can clearly read what action they are about to perform and the cognitive overhead is pretty simple:
- "Star dashboard"
- "Unstar dashboard"

However, for a screen reader user it becomes more complicated when the element has state. In this case they would get presented with:

- "Unstar dashboard, ticked"
- "Star dashboard, unticked"

So it becomes more difficult to reason what will happen and for them to decipher state. MDN has a section on this  under [aria-pressed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed).